lmdeploy/pytorch/check_env/__main__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Can we reuse `lmdeploy check_env`?

Reply:
Nope, I need a usability check instead of version check.